service/src/main/java/bio/terra/pipelines/db/entities/PipelineNameConverter.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.pipelines.db.entities;
+
+import bio.terra.pipelines.common.utils.PipelinesEnum;
+import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
+
+// inspired by https://www.baeldung.com/jpa-persisting-enums-in-jpa
+@Converter(autoApply = true)
+public class PipelineNameConverter implements AttributeConverter<PipelinesEnum, String> {
```

Comment:
now that we're getting a number of these converter classs, can we put them in their own directory under `/db/` or `/db/entities/`